From 5ac8dc185267518a30fb79a4b48c13ea47ab6a42 Mon Sep 17 00:00:00 2001
From: "Dr. Greg" <greg@enjellic.com>
Date: Thu, 11 Feb 2021 18:43:45 -0600
Subject: [PATCH 27/50] Change nomenclature for point display function.

This update removes the use of a variable named identity in the
function that locates a point on the event domain and displays
it and replaces it with a variable named point which is
consistent with the nomenclature change being implemented.
---
 security/integrity/ima/ima_events.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/security/integrity/ima/ima_events.c b/security/integrity/ima/ima_events.c
index 99987c3c5c8b..09b48fac17fb 100644
--- a/security/integrity/ima/ima_events.c
+++ b/security/integrity/ima/ima_events.c
@@ -921,11 +921,11 @@ static void ima_point_stop(struct seq_file *c, void *pos)
 	mutex_unlock(&map->point_list_mutex);
 }
 
-static int ima_point_show(struct seq_file *c, void *identity)
+static int ima_point_show(struct seq_file *c, void *point)
 {
 	struct event_point *id;
 
-	id = list_entry(identity, struct event_point, list);
+	id = list_entry(point, struct event_point, list);
 	seq_printf(c, "%*phN\n", WP256_DIGEST_SIZE, id->point);
 	return 0;
 }
-- 
2.31.1

